integration-cli/docker_cli_push_test.go

```diff
@@ -64,9 +59,7 @@ func (s *DockerRegistrySuite) TestPushMultipleTags(c *check.C) {
 
 	// Ensure layer list is equivalent for repoTag1 and repoTag2
 	out1, _ := dockerCmd(c, "pull", repoTag1)
-	if strings.Contains(out1, "Tag t1 not found") {
-		c.Fatalf("Unable to pull pushed image: %s", out1)
-	}
+
```

Comment:
I can confirm what @vdemeester  says, it will fail the `dockerCmd` if failed to pull the image.